src/app-layer-ssl.h

```diff
@@ -176,9 +178,17 @@ typedef struct SSLState_ {
 
     /* there might be a better place to store this*/
     uint16_t hb_record_len;
+
+    uint8_t finished;
+
+    DetectEngineState *de_state;
+
+    uint16_t events;
```

Comment:
the layout here can be made more efficient, this will waste 6 bytes between the 2 pointers